src/arch/devlog.cpp

```diff
@@ -98,7 +100,13 @@ void logMessage(E_MsgLevel paLevel, const char *paMessage, ...) {
 
 void printLogMessage(E_MsgLevel paLevel, const char *paMessage) {
 #ifndef __ZEPHYR__
-  (paLevel == E_MsgLevel::Error ? std::cerr : std::cout) << scLogLevel[static_cast<int>(paLevel)] << ": " << getNanoSecondsMonotonic() << ": " << paMessage;
+#ifdef FORTE_LOGGER_TIME_IN_DT
```

Comment:
sorry to be so direct but this is unreadable code. Can you please create a method like `getTimeString` and have then two C++ files:
  - `plainTimeString.cpp` implementing `getTimeString` with  simple time string 
  - `readableTimeString.cpp` implementing `getTimeString` using func_NOW

in CMakeList.txt you can then add the right cpp file based on the CMake option. 

So the code here does not need another define.

Reply:
Done.